lib/src/clixon_uid.c

```diff
@@ -173,6 +173,8 @@ drop_priv_temp(uid_t new_uid)
 #ifdef HAVE_GETRESUID
     int retval = -1;
     
+    clicon_debug(4, "%s uid:%u", __FUNCTION__, new_uid);
```

Comment:
Please use constants here:  https://github.com/clicon/clixon/blob/1698c6717e56f8fd298c2bafe8304a69cd0baa95/lib/clixon/clixon_log.h#L55

Reply:
You're aware that there are a lot of places where the constants aren't being used, right?

Reply:
yes for sure, the whole masking mechanism was introduced in 2023, but I try to follow it in new code